packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -279,6 +282,12 @@ const methods = {
     log("Method: ui called", uiConfig);
     const stylesConfig = uiConfig.styles;
 
+    if (stylesConfig) {
```

Comment:
As discussed in thread, I find the CSS variables approach the best approach. On requirement basis, we can keep creating the css variables and use them. It ensures that any supported UI customization is explicitly handled in the code and thus avoids regression.

Reply:
https://threads.com/34462876572?s=c3wvA7ob5mRkwkg6je4CTk